front/src/styles/scss/common/components/_manchetteWithSpaceTimeChart.scss

```
@@ -0,0 +1,26 @@
+.manchette-space-time-chart-wrapper{
+    border-radius: .625rem;
+    box-shadow: 0rem .25rem .5625rem rgba(0, 0, 0, 0.06), 0rem .0625rem .125rem rgba(0, 0, 0, 0.19);
+    background-color: rgba(255, 255, 255, 1);
+
+
+    .header {
+      height: 2.5rem;
+      background-color: var(--ambiantB5);
+      border-color: var(--black25);
+      border-bottom-width: .0625rem;
+box-shadow: 0px 1px 0px rgba(255, 255, 255, 1) inset;
```

A bit strange I give you that but since it's in the design system :)
You can also convert pixels into rems in the new box-shadow

```suggestion
    background-color: var(--white100);


    .header {
      height: 2.5rem;
      background-color: var(--ambiantB5);
      border-color: var(--black25);
      border-bottom-width: .0625rem;
box-shadow: 0px 1px 0px var(--white100) inset;
```